apps/token/src/routes/governance/proposal/proposal-container.tsx

```diff
@@ -36,16 +19,7 @@ export const PROPOSAL_QUERY = gql`
 `;
 
 export const ProposalContainer = () => {
-  const { t } = useTranslation();
   const params = useParams<{ proposalId: string }>();
-  const proposalUrl = React.useMemo(
-    () => new URL(`governance/proposal/${params.proposalId}`, ENV.restUrl).href,
```

Comment:
chore (blocking?): i think it would be more future proof (well, less confusing in the future) if we'd remove this restUrl from the ENV object, and remove the whole `NX_VEGA_REST` from all the env files.

Reply:
The rest url still appears to be used in some circumstances. It goes beyond the reach of this quick-fix ticket. I'll make a separate ticket to go through and remove the env var where possible.